Scripts/script-ConvertXmlFileToJson.yml

```diff
@@ -0,0 +1,36 @@
+commonfields:
+  id: ConvertXmlFileToJson
+  version: -1
+name: ConvertXmlFileToJson
+script: |-
+  var xml = getFileByEntryID(args.entryID);
+  var xmlJson = x2j(xml);
+  try {
+    xmlJson = JSON.parse(xmlJson);
+  } catch(err) {}
+
+  if (args.contextKey && args.contextKey.length > 0) {
+      setContext(args.contextKey,xmlJson);
+  }
+  if (args.verbose && args.verbose.length > 0 && args.verbose == 'True'){
+      return xmlJson;
```

Comment:
I think we should set a default value to this otherwise running it without anything wouldn't return results

Reply:
Yeah, I think the same but the original script did not so I thought maybe there is some smarts here I don't understand...

Reply:
Will fix both scripts